Library/Formula/git-plus.rb

```diff
@@ -0,0 +1,25 @@
+class GitPlus < Formula
+  homepage "https://github.com/tkrajina/git-plus"
+  head "https://github.com/tkrajina/git-plus.git"
+  url "https://github.com/tkrajina/git-plus/archive/v0.1.0.tar.gz"
+  sha256 "d7076ed3fd8e6e4bc2b93b9880adf58bea59b0f6fe95e05053f00247ae8cb609"
+
+  depends_on :python if MacOS.version <= :snow_leopard
+
+  def install
+    site_packages = libexec/"lib/python2.7/site-packages"
+    libexec_bin = libexec/"bin"
+
+    ENV.prepend_create_path("PYTHONPATH", site_packages)
+
+    site_packages.install("gitutils.py")
+    libexec_bin.install(Dir["git-*"])
+
+    bin.install(Dir[libexec_bin/"*"])
+    bin.env_script_all_files(libexec_bin, :PYTHONPATH => ENV["PYTHONPATH"])
+  end
+
+  test do
+    shell_output("git multi")
```

Comment:
No, the example needed a hyphen between the command. As in:

```ruby
system bin/"git-multi"
```

That is how it is installed in the ` bin `:

```
ln -s ../Cellar/git-plus/0.1.0/bin/git-multi git-multi
ln -s ../Cellar/git-plus/0.1.0/bin/git-old-branches git-old-branches
ln -s ../Cellar/git-plus/0.1.0/bin/git-relation git-relation
```